.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: "\U0001F680 Feature request"
+about: Suggest a new feature or a big change
+
+---
+<!-- Thank you for contributing. These HTML commments will not render in the issue, but you can delete them once you've read them if you prefer! -->
+
+### Proposed change
+<!-- Use this section to describe the feature you'd like to be added. -->
+
+
+### Alternative options
+<!-- Use this section to describe alternative options and why you've decided on the proposed feature above. -->
+
+
+### Who would use this feature?
+<!-- Describe the audience for this feature. This information will affect who chooses to work on the feature with you. -->
+
+
+### (Optional): Suggest a solution
+<!-- Describe what you think needs to be done. This information will affect who chooses to work on the feature with you -->
+
+
+### Who can do this work?
+<!-- What skills are needed? Who can be recruited to add this feature? This information will affect who chooses to work on the feature with you. -->
```

Comment:
```suggestion
```

I think the _who_ part is problematic unless the response is "I can do it". I think the _what skills_ part could be relevant, but I think it can follow from the suggest a solution part above.